arch/arm/src/dm320/dm320_decodeirq.c

```diff
@@ -109,11 +102,9 @@ uint32_t *arm_decodeirq(uint32_t *regs)
             }
 #endif
 
-          /* Set current_regs to NULL to indicate that we are no longer in
-           * an interrupt handler.
-           */
+          /* Set irq flag */
 
-          up_set_current_regs(NULL);
+          arm_set_irq_flag(false);
```

Comment:
why not keep the orignal style? Will `bool` have better performance than `uint32_t`?

Reply:
The size of bool is smaller, and its meaning is more clearer here

Reply:
Could we keep the arg type as orignal?

Reply:
but we don't save pointer value, why continue use NULL?